doc/userguide/setting-up-ipsinline-for-linux.rst

```diff
@@ -117,6 +117,14 @@ The options -i (input) -o (output) can be combined with all previous mentioned o
 
 If you would stop Suricata and use internet, the traffic will not come through. To make internet work correctly, you have to erase all iptable rules.
 
+To simply inspect all traffic on the host, before routing desicion and input/output separation, you can use mangle PREROUTING table:
+
+::
+
+
+  sudo iptables -t mangle -I PREROUTING -j NFQUEUE
```

Comment:
This is wrong: this setup only works for FORWARDED traffic. In the case of INPUT/OUTPUT we will only get the INPUT traffic. Look at the output of iptables after doing a SSH from the local box to an external host.
```
iptables -L -nv -t mangle
Chain PREROUTING (policy ACCEPT 273 packets, 287K bytes)
 pkts bytes target     prot opt in     out     source               destination         
   18  4627            tcp  --  *      *       0.0.0.0/0            0.0.0.0/0            tcp spt:22
    0     0            tcp  --  *      *       0.0.0.0/0            0.0.0.0/0            tcp dpt:22
``` 
We don't see the packet in the output direction.